tests/services/document-fetcher.test.js

```diff
@@ -84,7 +84,10 @@ describe('DocumentFetcher', () => {
     const expectedPredicatesAtValue = Symbol('AT')
     const uid = Symbol('UID')
     const prismicApi = {
-      query: jest.fn(() => ({ results: [expectedValue] })),
+      query: jest
+        .fn()
+        .mockImplementationOnce(() => ({ results: [expectedValue] }))
+        .mockImplementationOnce(() => ({ results: [] })),
```

Comment:
Ça donnerait quelque chose comme ça je pense : 
```suggestion
        .mockImplementationOnce(getSimplePageByUidMock)
        .mockImplementationOnce(getSlicesPageByUidMock),
```